### ISSUE TITLE
Add Disaster Data

### ISSUE THREAD

**Issue by mdsimpson42** [colouring-cities/colouring-core#1138]
Add 'Disaster Management' Section to Resilience Category.

- Disaster Type
- Severity
- Assessment Method
- Source(s)

**Comment by mdsimpson42** [comment]
The initial development has been merged onto the ui-refactor branch. Once that is merged onto main, I will close this issue and any new features/bugs etc. can be added as new issues

**Comment by mdsimpson42** [comment]
Most of this is now done, except for the dates, which is a separate issue #1187
